llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -84,8 +84,12 @@ RISCVTTIImpl::getRISCVInstructionCost(ArrayRef<unsigned> OpCodes, MVT VT,
       Cost += VL;
       break;
     }
+    case RISCV::VMV_X_S:
+    case RISCV::VFMV_F_S:
     case RISCV::VMV_S_X:
-      // FIXME: VMV_S_X doesn't use LMUL, the cost should be 1
+    case RISCV::VFMV_S_F:
+      Cost += 1;
+      break;
```

Comment:
Can we split this off into a separate PR? I think it would be useful to be able to see if this has any effect on the current cost model tests.

Reply:
Sure, I split it in #78739. will update this PR after that.